fe/fe-core/src/main/java/com/starrocks/planner/StreamLoadScanNode.java

```diff
@@ -160,7 +160,7 @@ public void init(Analyzer analyzer) throws UserException {
             String sep = streamLoadTask.getColumnSeparator().getColumnSeparator();
             byte[] setBytes = sep.getBytes(StandardCharsets.UTF_8);
             params.setColumn_separator(setBytes[0]);
-            if (sep.length() > 50) {
+            if (setBytes.length > 50) {
                 throw new UserException("the column separator is limited to a maximum of 50 bytes");
```

Comment:
the row delimiter is limited to a maximum of 50 bytes

Reply:
Here is the column separator